src/fondant/core/schemas/manifest.json

```diff
@@ -60,13 +60,13 @@
       },
```

Comment:
The `field` definition here has a `fields` attribute, which I don't think is correct.